Cargo.toml

```diff
@@ -32,6 +32,11 @@ thiserror = "1.0.30"
 lazy_static = "1.4.0"
 tracing = "0.1.37"
 tracing-subscriber = "0.3.17"
+opentelemetry-appender-tracing = { version = "0.27.0", default-features = false}
```

Comment:
note: I only added the system to forward the `tracing` crate to opentelemtry, but not other crates

that is to say, I noticed in Dolos (and maybe downstream crates?) you occasionally use `use log::` which won't be forwarded